enhancements/installer/assisted-installer.md

```diff
@@ -0,0 +1,372 @@
+shown in the UI.
+
+### Disconnected Deployments
+
+The disconnected process is similar to the connected process, except
+that the assisted installer application must run somewhere in the
+disconnected environment, and the agents need to be directed there
+instead of to the cloud. There are several open questions related to
+how this would work.
+
+### Integration with Existing Bare Metal IPI Tools
+
+Clusters built using the assisted installer workflow use the same
+"baremetal" platform setting as clusters built with the IPI
+installer. The cluster runs metal3, without PXE booting support.
+
+BareMetalHosts created by the assisted installer workflow do not have
+BMC credentials set. This means that power-based fencing is not
+available for the associated nodes until the user provides the BMC
+details.
+
+### User Stories
+
+#### Story 1
+
+As a cluster deployer, I want to install OpenShift on a small set of
+hosts without having to make configuration changes to my network or
+obtain administrator access to infrastructure so I can experiment
+before committing to a full production-quality setup.
+
+#### Story 2
+
+As a cluster deployer, I want to install OpenShift on a large number
+of hosts using my existing provisioning tools to automate launching
+the installer so I can adapt my existing admin processes and
+infrastructure tools instead of replacing them.
+
+#### Story 3
+
+As a cluster deployer, I want to install a production-ready OpenShift
+cluster without committing to delegating all infrastructure control to
+the installer or to the cluster, so I can adapt my existing admin
+processes and infrastructure management tools instead of replacing
+them.
+
+#### Story 4
+
+As a cluster hardware administrator, I want to enable power control
+for the hosts that make up my running cluster so I can use features
+like fencing and failure remediation.
+
+### Implementation Details/Notes/Constraints
+
+Much of the work described by this enhancement already exists as a
+proof-of-concept implementation. Some aspects will need to change as
+part of moving from PoC to product. At the very least, the code will
+need to be moved into more a suitable GitHub org.
+
+The agent discussed in this design is different from the
+`ironic-python-agent` used by Ironic in the current IPI
+implementation.
+
+### Risks and Mitigations
+
+The current implementation relies on
+[minimise-baremetal-footprint](../baremetal/minimise-baremetal-footprint.md). If
+that approach cannot be supported, users can proceed by providing an
+extra host (4 hosts to build a 3 node cluster, 6 hosts to build a 5
+node cluster, etc.).
+
+Connected deployments rely on services running on
+`cloud.redhat.com`. Many corporate networks require firewall or proxy
+settings to be able to access outside services. We will provide a way
+to run the same services locally for cases where remote "connected"
+access is not available.
+
+## Design Details
+
+### Open Questions
```

Comment:
Are there any other open questions to include in this draft before publishing it next week?